app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -16,10 +16,11 @@ import kotlinx.android.synthetic.main.content_no_internet.view.noInternetCard
 import kotlinx.android.synthetic.main.content_no_internet.view.retry
 import kotlinx.android.synthetic.main.fragment_events.eventsNestedScrollView
 import kotlinx.android.synthetic.main.fragment_events.view.eventsRecycler
-import kotlinx.android.synthetic.main.fragment_events.view.homeScreenLL
-import kotlinx.android.synthetic.main.fragment_events.view.locationTextView
+import kotlinx.android.synthetic.main.fragment_events.view.noEventsMessage
 import kotlinx.android.synthetic.main.fragment_events.view.progressBar
 import kotlinx.android.synthetic.main.fragment_events.view.shimmerEvents
+import kotlinx.android.synthetic.main.fragment_events.view.homeScreenLL
+import kotlinx.android.synthetic.main.fragment_events.view.locationTextView
```

Comment:
Why these changes?

Reply:
sorry for that, I will make sure I learn how to check for difference of files carefully before making a push and PR, I fixed that